Proton/Sources/ObjC/include/PRTextStorage.h

```diff
@@ -25,21 +25,35 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@protocol DefaultTextFormattingProviding;
-@protocol TextStorageDelegate;
-@class Attachment;
+NS_SWIFT_NAME(DefaultTextFormattingProviding)
+@protocol PRDefaultTextFormattingProviding<NSObject>
+@optional
+@property UIFont *font;
+@property NSMutableParagraphStyle *paragraphStyle;
+@property UIColor *textColor;
```

Comment:
Probably non-mutable, right?
```suggestion
@property (nonatomic, readonly) UIFont *font;
@property (nonatomic, readonly) NSMutableParagraphStyle *paragraphStyle;
@property (nonatomic, readonly) UIColor *textColor;
```